reference/orders.v2.oas2.yml

```diff
@@ -136,6 +136,29 @@ paths:
                       country: United States
                       country_iso2: US
                       email: elsie@example.com
+                  consignments:
+                    addresses:
+                      address_1: '125 Lane'
+                      address_2: '108 Lane'
+                      city: Austin
+                      state: Texas
+                      zip: '78108'
+                      country: United States
+                      country_iso2: US
+                      phone: '1234567890'
+                      email: janedoe@example.com
+                      customfields:
+                        length: 0
+                    couponDiscounts: 0
+                    discounts: 
+                    handlingcost: 0
+                    id: 65c53b429a0ab
+                    lineitemIds: '138e9a16-1a51-43c7-b4d6-6d8cd9f62189'
+                    selectedShippingOption: null
+                    shippingAddress:
+                      url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/247/shippingaddresses'
+                      resource: /orders/247/shippingaddresses
+                    shippingCost: 0
```

Comment:
This doesn't seem like a valid request body?
Have a look at https://github.com/bigcommerce/orders/blob/bae2a45bd4de9c014d421e7c1ef3a091eaf0eab2/tests/api/Consignment/MixedConsignmentCest.php#L52 for an example of the structure for our requests. That is for a `POST` but the `PUT` should be the same.
This consignment would need to be wrapped in a `{consignments: { shipping: [...]}}` structure.

Also the example already includes `products`. Our requests should be using `products` or `consignments` but not both.

Reply:
You're right, it was a ball of wrong. I added Order with Consignments to the examples dropdown. I used the data from the link you sent.